kubernetes/template/DeploymentTemplates/createUiDefinition.json

```diff
@@ -749,6 +749,17 @@
               "required": true
             },
             "visible": false
+          },
+          {
+            "name": "disconnectedAKSeURL",
+            "type": "Microsoft.Common.TextBox",
+            "label": "AKS Engine Base url (Disconnected)",

```

Comment:
`URL` (uppercase)

Reply:
I think you should just call it localAKSeBinaryURL
Also in the label and tooltip add binary. In the tool tip you should also add a link to the page where we list the support binaries for AzureStack